File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_2/exercice09.c
#include <stdio.h>
int n;
float facture;
int main(){
    printf("saisir le nombre des photocopies : ");
    scanf("%d",&n);
    if(n<=10)
        facture=n*0.1;
    else if(n>10 && n<-30)
        facture=10*0.1+(n-10)*0.09;
    else
        facture=10*0.1+20*0.09+(n-10)*0.08;
    printf("la facture est : %.3f\n",facture);
    return 0;
}

File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_2/exercice09.py
n=int(input("saisir le nombre des photocopies : "))
if(n<=10):
    facture=n*0.1
elif(n>10 and n<-30):
    facture=10*0.1+(n-10)*0.09
else:
    facture=10*0.1+20*0.09+(n-10)*0.08
print("la facture est : ",facture)